config/crd/external/kustomization.yaml

```diff
@@ -1,3 +1,9 @@
 resources:
 - serving.kserve.io_inferenceservices.yaml
-# - route.openshift.io_routes.yaml  #minikube
+- serving.kserve.io_servingruntimes.yaml
+- route.openshift.io_routes.yaml
+- monitoring.coreos.com_podmonitors.yaml
+- monitoring.coreos.com_servicemonitors.yaml
+- maistra.io_servicemeshmemberrolls.yaml
+- maistra.io_servicemeshmembers.yaml
+- telemetry.istio.io_telemetries.yaml
```

Comment:
are all these resources needed?

Reply:
Actually I had to add all of them to make e2e tests working in the Kind cluster (i.e., they are used just for `deploy-dev` and `deploy-e2e`)  - not sure if we want to move them directly in the overlay/e2e config folder

Reply:
@lampajr we have a the `overlays/dev` directory for this purpose (for local testing etc). I think adding these CRDs here is not necessary and IMO if these are needed to make local testing work they should be added to the `overlays/dev/kustomization.yaml` file and apply that overlay for local testing

Reply:
@lampajr looking at the manifests again, if the `crd/external` directory is not in the call chain for the actual deployment, then it's fine to add these here. We just want to avoid applying these CRDs on the cluster outside of development testing scenarios.